File: src/main/java/nearbyearthquakes/domain/Earthquake.java
package nearbyearthquakes.domain;

/**
 * Representation of the Earthquake in the app
 */
public class Earthquake {

    private final String title;
    private final double distance;
    private final double latitude;
    private final double longitude;

    public Earthquake(String title, double distance, double latitude, double longitude) {
        this.title = title;
        this.distance = distance;
        this.latitude = latitude;
        this.longitude = longitude;
    }

    public String getTitle() {
        return title;
    }

    public double getDistance() {
        return distance;
    }

    public double getLatitude() {
        return latitude;
    }

    public double getLongitude() {
        return longitude;
    }

    @Override
    public String toString() {
        return title + " || " + (int) distance;
    }
}

File: src/domain/Earthquake.js
// Give a little bit of life to the result by adding some colour ( lets have a some fun)
const chalk = require('chalk');

/**
 * Representation of Earthquake in the domain app
 */
module.exports = class Earthquake {

    constructor(title, coordinates, distance) {
        this.title = title;
        this.coordinates = coordinates;
        this.distance = distance;
    }

    /**
     * print to the console in the formated representation asked ( and with some colour)
     */
    printToConsole = () => console.log(chalk.red(this.title), chalk.yellow(" || "), chalk.green(this.distance));

}